doc/en/components/grids/_shared/cell-selection.md

```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
 
 Below are the methods that you can use in order to select ranges, clear selection or get selected cells data.
 
-<!-- Angular -->
+
+<!-- Angular, WebComponents -->
+
+### Select range
+
+`selectRange` - Select a range of cells with the API. rowStart and rowEnd should use row indexes and columnStart and columnEnd could use column index or column data field value.
+
+```ts
+const range = { rowStart: 2, rowEnd: 2, columnStart: 1, columnEnd: 1 };
+this.grid.selectRange(range);
+```
+
+<!-- end: Angular, WebComponents -->
+
 ### Get Selected Data
 
-`GetSelectedData` will return array of the selected data in format depending on the selection. Examples below:
+<!-- Blazor -->
+`GetSelectedData` will return array of the selected data in Dictionary format. Examples below:
+
+```razor
+<IgbGrid @ref=grid  CellSelection="GridSelectionMode.Multiple" AutoGenerate=true></IgbGrid>
+
+@code {
+    private IgbGrid grid;
+
+    private async void GetSelectedData()
+    {
+        object[] data = await this.grid.GetSelectedDataAsync(true, true);
+    }
+}
+```
+<!-- end: Blazor -->
+
+<!-- Angular, WebComponents -->
+`getSelectedData` will return array of the selected data in format depending on the selection. Examples below:
 
 1. If three different single cells are selected:
```

Comment:
Those are shown not as list 
![Screenshot 2023-05-16 at 14 03 26](https://github.com/IgniteUI/igniteui-xplat-docs/assets/37667452/9ceff458-cffa-429d-8484-7cdab09fdda8)

Reply:
Formatting should be fixed now.